docs/source/ucx.rst

```diff
@@ -0,0 +1,122 @@
+UCX Integration
+===============
+
+Communication can be a major bottleneck in distributed systems, and that's no different in Dask and Dask-CUDA. To address that, Dask-CUDA provides integration with `UCX-Py <https://ucx-py.readthedocs.io/>`_, a Python interface for the `UCX <https://www.openucx.org/>`_ communication framework. UCX is a low-level library that provides high-performance networking, supporting several transports, including NVLink and InfiniBand for systems that have such specialized hardware, as well as TCP for those that do not.
+
+
+Relevant Settings
+-----------------
+
+In this list below we shall see all the Dask configurations relevant for using with Dask-CUDA currently. Note that these options can also be used with mainline Dask/Distributed if you can't use Dask-CUDA for some reason. Note that many improvments for running GPU code is readily available in Dask-CUDA -- but not in mainline Dask/Distributed -- thus, we recommend using Dask-CUDA when possible.
```

Comment:
```suggestion
In the list below we describe the relevant Dask configurations for using UCX with Dask-CUDA.  Note: these options can also be used with mainline Dask/Distributed outside of Dask-CUDA, however, we recommend using Dask-CUDA when possible.
```

Reply:
I want to keep the information about the advantages of using Dask-CUDA, removing that to just say "we recommend that" seems like it's just because we want to force that upon people for no reason, which is far from true. For that reason, I added another section describing benefits of using Dask-CUDA. I'm still looking at how to cross-reference sections, but will shortly push a change for that.